documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -329,11 +329,11 @@ NOQuuuWuFMMMCCCCPPPPTTTTTTTTRRRR
 
 ^1^ Information on xref:raspberry-pi.adoc#otp-register-and-bit-definitions[programming the OTP bits].
 
-^2^ Warranty bit is never set on Pi4.
+^2^ Warranty bit is never set on Pi 4.
 
-=== New-style revision Codes In Use
```

Comment:
Title case has "in" as lower case (since it is less than 4 letters)

Reply:
You've commented on a deleted line in red. This was actually what prompted me to file the PR, since it was obviously wrong. The line below it in green shows I've already fixed this.